pkg/source/markdown.go

```diff
@@ -108,12 +114,22 @@ func (m *markdown) parse(filename string, opts Opts) (err error) {
 			count++
 		}
 
-		if count == 1 && !doxIdFound {
-			found, err := fmt.Sscanf(line, m.escape(doxIdFmt), &m.id)
+		if count == 1 || count == 2 {
+			if !doxIdFound {
+				found, err := fmt.Sscanf(line, m.escape(doxIdFmt), &m.id)
 
-			if err == nil && found > 0 {
-				doxIdFound = true
-				continue
+				if err == nil && found > 0 {
+					doxIdFound = true
+					continue
+				}
+			}
+
+			if !doxIgnoreFound {
+				if strings.Contains(line, m.escape(doxIgnore)) {
+					m.ignore = true
+					doxIgnoreFound = true
+					continue
```

Comment:
If this `Source` is ignored, do we need to continue parsing?

Reply:
- [ ] if ignored, break out of loop